ui/src/containers/NodePageMetricsTab.js

```diff
@@ -147,36 +278,46 @@ const NodePageMetricsTab = (props) => {
     title: null,
   };
 
-  const colorUsage = {
-    field: 'type',
+  const strokeDashConfig = {
+    field: 'symbol',
     type: 'nominal',
-    legend: null,
-    domain: ['y'],
-    scale: {
-      range: ['#4BE4E2'],
+    legend: {
+      direction: 'horizontal',
+      orient: 'bottom',
+      title: null,
+      values: ['Cluster avg'],
+      symbolSize: 300,
+      labelFontSize: 15,
     },
   };
-  const colorSystemLoad = {
-    field: 'type',
-    type: 'nominal',
-    legend: null,
-    domain: ['y'],
-    scale: {
-      range: ['#A6B561'],
+
+  const opacityConfig = {
+    condition: {
+      test: 'datum.symbol == "Cluster avg"',
+      value: 0.5,
     },
+    value: 1,
   };
-  const colorMemory = {
+
+  // the `read` and `out` should be the same color
+  // the `write` and `in` should be the same color
+
+  const colorUsage = {
     field: 'type',
     type: 'nominal',
-    legend: null,
-    domain: ['y'],
-    scale: {
-      range: ['#1F78C1'],
-    },
+    scale: { range: ['#9645c2'] },
+    legend: showAvg
+      ? {
+          direction: 'horizontal',
+          orient: 'bottom',
+          title: null,
+          values: ['%'],
+          labelFontSize: 15,
+          symbolSize: 300,
+        }
+      : null,
```

Comment:
Instead of null display empty legend would solve the charts moving issue
```suggestion
     : {
          direction: 'horizontal',
          orient: 'bottom',
          title: '',
          values: [''],
          labelFontSize: 15,
          symbolSize: 300,
        },,
```